overlore/constants.py

```diff
@@ -1,4 +1,7 @@
 # NPC PARAMS
 SEX = ["male", "female"]
-ROLES = ["farmer", "miner", "fisherman", "merchant", "trader", "soldier"]
+ROLES = ["farmer", "miner", "fisherman", "merchant", "soldier"]
 TRAIT_TYPE = ["positive", "negative"]
+
+MIN_AGE = 15
+MAX_AGE = 65
```

Comment:
Move these constants to the file they're used in as they only have a local purpose